data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -348,6 +348,44 @@ then
     record_service_stage_success
 fi
 
+if [ "$BOOTSTRAP_INPLACE" = true ] && [ ! -f cmdbootline-nto-bootstrap.done ]; then
+	record_service_stage_start "cmdbootline-nto-bootstrap"
+
+	rm --recursive --force cmdbootline-nto-bootstrap
+
+	# Create folders for overlay and podman command to use them
+	tmpfolder=$(mktemp -d)
+	echo ${tmpfolder}
+	podmancmd="bootkube_podman_run --attach=stderr --rm "
+	for folder in var/lib/tuned/profile-data etc/modprobe.d etc/sysconfig etc/kubernetes etc/sysctl.d etc/systemd run sys lib/modules var/lib/kubelet usr/lib/tuned etc/tuned; do
```

Comment:
@jmencak  @jlojosnegros   do we need to align with https://github.com/openshift/cluster-node-tuning-operator/pull/970  in some manner ? or that change is not affecting the installer ?

Reply:
Good point. It looks like it needs an update. Also note that `/var/lib/tuned/profiles-data` was removed, not sure what `var/lib/tuned/profile-data` (singular) is used for. @jlojosnegros typo?

Reply:
yeah it is a typo